lib/gyro/infos.rb

```diff
@@ -0,0 +1,29 @@
+# Gyro Infos
+#
+module Gyro	
+	module Infos
+		TEMPLATE_FILE = "./lib/templates"
```

Comment:
Don't hardcode this with a relative path. It won't work if you execute gyro from somewhere outside its directory (especially once gyro is installed as a gem)

There's a method we created in gyro:liquid:generator to reference the templates directory (we use it to _load_ the template we want to use); use that method here to find the templates here instead of that constant